test/interactions/clickInteractionTests.ts

```diff
@@ -1,158 +1,208 @@
 ///<reference path="../testReference.ts" />
 
-describe("Interactions", () => {
-  describe("Click Interaction", () => {
-    describe("Basic Usage", () => {
-      let SVG_WIDTH = 400;
-      let SVG_HEIGHT = 400;
+describe("Click Interaction", () => {
+  const SVG_WIDTH = 400;
+  const SVG_HEIGHT = 400;
```

Comment:
One thing that we are recently noticing in our test refactors is to less rely on setting the svg width / height to some static value, since the way of generating the svg comes with a default width/height and we can avoid some of the clutter of setting a width/height every single time.

Would it make more sense to remove these variables?  Either that or create variables that would be set in the `beforeEach` that would grab the values of the width of the svg or the height?